src/main/java/com/linkedin/kmf/services/MultiClusterTopicManagementService.java

```diff
@@ -296,15 +324,16 @@ void maybeAddPartitions(int minPartitionNum) {
     }
 
     void maybeReassignPartitionAndElectLeader() throws Exception {
-      KafkaZkClient zkClient = KafkaZkClient.apply(_zkConnect, JaasUtils.isZkSecurityEnabled(), ZK_SESSION_TIMEOUT_MS,
-          ZK_CONNECTION_TIMEOUT_MS, Integer.MAX_VALUE, Time.SYSTEM, METRIC_GROUP_NAME, "SessionExpireListener");
+      KafkaZkClient zkClient = KafkaZkClient.apply(_zkConnect, JaasUtils.isZkSecurityEnabled(), com.linkedin.kmf.common.Utils.ZK_SESSION_TIMEOUT_MS,
```

Comment:
Can this be done with AdminClient.electPreferredLeaders?

Reply:
bumped up the dependency on kafka-clients in order to leverage the AdminClient exposed electPreferredLeaders.